core/src/main/resources/jenkins/monitor/EndOfLifeAdminMonitor/message.jelly

```diff
@@ -0,0 +1,45 @@
+<!--
+The MIT License
+
+Copyright (c) 2021 Tim Jacomb.
+Copyright (c) 2023 Mark Waite.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:f="/lib/form">
+    <div class="alert alert-danger">
+        <form id="${it.id}" method="post" action="${rootURL}/${it.url}/act" name="${it.id}">
+            <f:submit name="yes" value="${%More Info}"/>
+            <l:isAdmin>
+                <f:submit value="${%Ignore} ${it.displayName}" name="no"/>
```

Comment:
The UI from the screenshots looks quite broken, buttons should not be that long and the heading shouldn't span multiple lines like that..
this may not be enough to fix it but it's a start
```suggestion
                <f:submit value="${%Dismiss}" name="no"/>
```

Reply:
Thanks.  Replaced long text with "Ignore" to be consistent with other admin monitors.  See 8ae1172bc1fead511aa33a5b8be2261e2ccbc654

Reply:
I could also remove the "and its derivatives" from the heading of the dialog box and rely on the administrator to read the body text in order to realize that their operating system is also end of life, even though it is not named "Red Hat Enterprise Linux".

Reply:
would it be better to put the actual OS, that way there's less user confusion and the heading is a lot simpler / shorter ? 

e.g. 

```
➜  azure-vm-agents-plugin git:(key-based-auth) ✗ docker run -it ubuntu bash
root@622f1c9a01ce:/# cat /etc/os-release | grep ^NAME=
NAME="Ubuntu"
root@622f1c9a01ce:/#
exit
➜  azure-vm-agents-plugin git:(key-based-auth) ✗ docker run -it centos:7 bash
Unable to find image 'centos:7' locally
7: Pulling from library/centos
6717b8ec66cd: Pull complete
Digest: sha256:be65f488b7764ad3638f236b7b515b3678369a5124c47b8d32916d6487418ea4
Status: Downloaded newer image for centos:7
[root@f369fa8f3f9c /]# cat /etc/os-release | grep ^NAME=
NAME="CentOS Linux"
```

I had a look around and there's not a better way that I can see, Java `os.name` property only shows linux

Reply:
> I had a look around and there's not a better way that I can see, Java `os.name` property only shows linux

The contents of `/etc/os-release` that match the version number specification provide a good phrasing for the detected operating system.  In the test data it looks like this:

```
os-release-alpine-3.14:PRETTY_NAME="Alpine Linux v3.14"
os-release-alpine-3.15:PRETTY_NAME="Alpine Linux v3.15"
os-release-alpine-3.16:PRETTY_NAME="Alpine Linux v3.16"
os-release-alpine-3.17:PRETTY_NAME="Alpine Linux v3.17"
os-release-centos-7:PRETTY_NAME="CentOS Linux 7 (Core)"
os-release-debian-10:PRETTY_NAME="Debian GNU/Linux 10 (buster)"
os-release-fedora-36:PRETTY_NAME="Fedora Linux 36 (Container Image)"
os-release-fedora-37:PRETTY_NAME="Fedora Linux 37 (Container Image)"
os-release-oracle-7:PRETTY_NAME="Oracle Linux Server 7.9"
os-release-redhat-7:PRETTY_NAME="Red Hat Enterprise Linux Server 7.9 (Maipo)"
os-release-scientific-7:PRETTY_NAME="Scientific Linux 7.9 (Nitrogen)"
os-release-ubuntu-18.04:PRETTY_NAME="Ubuntu 18.04.6 LTS"
```

The definition of `/etc/os-release` states that version information in `PRETTY_NAME` is optional, but in all the cases noted here, it includes a version number.  All the test data files tracked in the platform labeler plugin also include a version number in the `PRETTY_NAME` value (except openSUSE Tumbleweed that is defined to not have a version number).